### DISCUSSION TITLE
Ideapad Flex 5-14IlL05 - 0.7.6

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/Lenovo-Ideapad-Flex-5-14IIL05_Hackintosh_OpenCore#4] [Announcements]
### News / Fixes:
- First Release, to my knowledge it should boot at least on recovery

This is Debug version in case things go wrong to fix them.

This will evolve over time be patient please.

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/Lenovo-Ideapad-Flex-5-14IlL05-Hackintosh/releases/tag/0.7.6-alpha'>Ideapad Flex 5-14IlL05 - 0.7.6</a>.</em>

**Comment by rhysbw** [comment]
Have you managed to make any progress with this at all? Was really looking forward to it when i found it. I'd love to test it but i don't really see the same type of instructions for this build as your other ones, any chance for those?

**Reply by RobyRew** [reply]
Im so sorry for this situation, I really wish to have more free time for this things, but unfortunately lately I have to work and go to my school ending up with all day "lost" and the weekends I study and do my homeworks.

I feel sorry about everything, I will try my best to give you and other people that are waiting for other guides an update, and in this Lenovo case fix those errors on the boot and maybe some BIOS configurations needs to be done.

I hope you are doing great there, Stay tuned for future news.

**Comment by rhysbw** [comment]
Thanks mate no worries. Ye i dont know if my computer is loading it wrong, it just all the images and stuff you have uploaded seem to either be removed or like the links link to something else. Anyways dont be sorry thank you:)
